#! /bin/sh -e
## gnome_applet_dir.dpatch by Samuel Mimram <samuel.mimram@ens-lyon.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian's "default" directory for applets.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/smimram/work/gnu/linphone/1.0/linphone-1.0.0pre4/gnome/GNOME_LinphoneApplet.server.in.in linphone-1.0.0pre4/gnome/GNOME_LinphoneApplet.server.in.in
--- /home/smimram/work/gnu/linphone/1.0/linphone-1.0.0pre4/gnome/GNOME_LinphoneApplet.server.in.in	2003-01-16 22:59:48.000000000 +0100
+++ linphone-1.0.0pre4/gnome/GNOME_LinphoneApplet.server.in.in	2004-10-25 11:45:01.000000000 +0200
@@ -2,7 +2,7 @@
 
 <oaf_server iid="OAFIID:GNOME_LinphoneApplet_Factory"
 	    type="exe"
-	    location="@LIBEXECDIR@/linphone_applet">
+	    location="/usr/lib/gnome-applets/linphone_applet">
 
 	<oaf_attribute name="repo_ids" type="stringv">
 		<item value="IDL:Bonobo/GenericFactory:1.0"/>
